src/services/opensrp/index.ts

```diff
@@ -70,4 +70,20 @@ export class OpenSRPService extends OpenSRPServiceWeb {
   ) {
     super(baseURL, endpoint, getPayload);
   }
+
+  public async readFile(
```

Comment:
1. Do we have a test for this?
2.  Can we add an issue to move this to OpenSRP web?

Reply:
https://github.com/onaio/reveal-frontend/issues/887